drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c

```diff
@@ -47,6 +47,7 @@
 #include <linux/list.h>
 #include <linux/of.h>
 #include <linux/platform_device.h>
+#include <linux/compat.h>
```

Comment:
I did do that in upstream, then got screamed at that I shouldn't have the CONFIG_COMPAT condition.